lib/Doctrine/Common/Collections/Criteria.php

```diff
@@ -61,6 +61,8 @@ public static function expr()
      * @param string[]|null $orderings
      * @param int|null      $firstResult
      * @param int|null      $maxResults
+     *
+     * @return void
```

Comment:
Yup

Reply:
That's not added for Psalm but in general, since no return type was in there before. Just thought I'dd add that too since I saw it while skimming the file. Can remove it though, if you don't want it in there! :)

Reply:
I don't have a strong opinion here. Generally, constructor return types look weird to me, even though it's possible to return values from a constructor in PHP.

Reply:
Oh wow, sorry. Didn't see we're talking about the constructor here. I agree that this shouldn't be added for a constructor. Removing it.